app/views/pets/show.html.erb

```diff
@@ -6,4 +6,5 @@
 <h4>Sex: <%= @pet.sex %></h4>
 <h4>Adoption Status: <%= @pet.adoption_status %></h4>
 <h4><a href='/pets/<%= @pet.id %>/edit'>Update <%= @pet.name %>'s information!</a></h4>
+<%=link_to "Delete #{@pet.name}'s whole existence! Go ahead!", "/pets/#{@pet.id}", :method => :delete%>
```

Comment:
`data-method="delete"` With `a href` you write it like so: `<a rel="nofollow" data-method="delete" href="/cart/2">Remove</a>`